libs/commons/File.mli

```diff
@@ -126,3 +126,12 @@ val erase_this_temp_file : Fpath.t -> unit
 (*****************************************************************************)
 val is_executable : Fpath.t -> bool
 val filesize : Fpath.t -> int
+
+(* [lines_of_file (start_line, end_line) file] returns
+ * the list of lines from start_line to end_line included.
+ *
+ * Note that the returned lines do not contain \n.
+ *
+ * This function is slow, you should not use it!
+ *)
+val lines_of_file : int * int -> Common.filename -> string list
```

Comment:
This module (which was created recently) should use only `Fpath.t` for file paths. We're moving away from `Common.filename` which is an alias for `string`. We haven't finished migrating all our code to `Fpath`, so if it's too inconvenient to use the fpath version of this function, move the non-fpath version to the `Common` module and have a wrapper using fpath in the `File` module.